src/version.js

```diff
@@ -30,6 +30,8 @@ const version = {
         // Update transaction weight constants
         transaction.updateTransactionWeightConstants(data.min_tx_weight, data.min_tx_weight_coefficient, data.min_tx_weight_k);
         tokens.updateDepositPercentage(data.token_deposit_percentage);
+        transaction.updateMaxInputsConstant(data.max_number_inputs);
```

Comment:
Shouldn't we increase the minimum API version to ensure it will work?

Reply:
Yes. I usually update this `MIN_API_VERSION` after merging the full node MR's and updating the `VERSION` field there. But could do it here, what do you think?

Reply:
I think we can do it here.

Reply:
Done